src/Tests/Seam/When_dispatching_messages_in_receive_context.cs

```diff
@@ -461,7 +461,7 @@ public class When_dispatching_messages_in_receive_context
         async Task<ITopologySectionManager> SetupEndpointOrientedTopology(TransportPartsContainer container, string enpointname, SettingsHolder settings)
         {
             container.Register(typeof(SettingsHolder), () => settings);
-            settings.SetDefault<EndpointName>(new EndpointName(enpointname));
+            settings.SetDefault("NServiceBus.Routing.EndpointName", enpointname);
```

Comment:
Ask the routing TF ;)

On Saturday, July 23, 2016, Sean Feldman <notifications@github.com> wrote:

> In src/Tests/Seam/When_dispatching_messages_in_receive_context.cs
> <https://github.com/Particular/NServiceBus.AzureServiceBus/pull/281#discussion_r71973194>
> :
>
> > @@ -461,7 +461,7 @@ public class When_dispatching_messages_in_receive_context
> >          async Task<ITopologySectionManager> SetupEndpointOrientedTopology(TransportPartsContainer container, string enpointname, SettingsHolder settings)
> >          {
> >              container.Register(typeof(SettingsHolder), () => settings);
> > -            settings.SetDefault<EndpointName>(new EndpointName(enpointname));
> > +            settings.SetDefault("NServiceBus.Routing.EndpointName", enpointname);
>
> @danielmarbach <https://github.com/danielmarbach> is EndpointName gone?
> Not a big fan of string keys...
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub
> <https://github.com/Particular/NServiceBus.AzureServiceBus/pull/281/files/10b73d2629f539224504c927421d8eb7f227c138#r71973194>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AAKossLv5WVJMtfSHg61IeE-G14Xu4cwks5qYibxgaJpZM4JSP_h>
> .
>